src/main/java/school/lemon/changerequest/java/employees/Employee.java

```diff
@@ -0,0 +1,51 @@
+package school.lemon.changerequest.java.employees;
+
+abstract public class Employee {
+
+    private static final int HOURS_PER_MONTH = 160;
+
+    protected double salary;
+    protected int hours;
+
+    public Employee(double salary, int hours) {
+        this.salary = salary;
+        this.hours = hours;
+    }
+
+    public Employee(double salary) {
+        this(salary, 0);
+    }
+
+    public Employee() {
+        this(0, 0);
+    }
+
+    public void setSalary(double salary) {
+        this.salary = salary;
+    }
+
+    public void addWorkHours(int hours) {
+        this.hours += hours;
+    }
+
+    public double getRatio() {
```

Comment:
What kind of ratio do you return? It's not obvious that it's about working hours.